test/assertions.test.ts

```diff
@@ -423,6 +423,43 @@ describe('runAssertion', () => {
 `,
   };
 
+  // is-sql assertions:
+  const isSQLAssertion: Assertion = {
```

Comment:
This is the tiniest of nits, but the convention in this codebase is to lowercase acronyms/initialisms in camel case var names.  For example, `isSqlAssertion`, `evalId`, `OpenAi`, etc...

Reply:
got it